atomicapp/providers/docker.py

```diff
@@ -54,37 +55,49 @@ def init(self):
             logger.info("DRY-RUN: Did not check Docker version compatibility")
         else:
             cmd_check = ["docker", "version"]
-            try:
-                docker_version = subprocess.check_output(cmd_check).split("\n")
-            except Exception as ex:
-                raise ProviderFailedException(ex)
-
-            client = ""
-            server = ""
-            for line in docker_version:
-                if line.startswith("Client API version"):
-                    client = line.split(":")[1]
-                if line.startswith("Server API version"):
-                    server = line.split(":")[1]
-
-            if client > server:
-                msg = ("Docker version in app image (%s) is higher than the one "
-                       "on host (%s). Please update your host." % (client, server))
-                raise ProviderFailedException(msg)
+
+            version = self.shipy(cmd_check)
+
+            compatibility = {
```

Comment:
would be great to not have to hardcode this. we should support whatever version docker-py supports right?